src/decode.rs

```diff
@@ -542,6 +543,19 @@ pub struct Dav1dFrameContext_bd_fn {
     pub backup_ipred_edge: backup_ipred_edge_fn,
     pub read_coef_blocks: read_coef_blocks_fn,
 }
+
+impl Dav1dFrameContext_bd_fn {
+    pub unsafe fn recon_b_intra(
+        &self,
+        context: *mut Dav1dTaskContext,
+        block_size: BlockSize,
+        flags: EdgeFlags,
+        block: *const Av1Block,
+    ) {
+        self.recon_b_intra.expect("non-null function pointer")(context, block_size, flags, block);
```

Comment:
A lot of these function pointers I think are never null.  Could the `Option` just be dropped instead?

Reply:
Wrote up #140 to track this.